tests/benchmark/test_worst_blocks.py

```diff
@@ -222,3 +230,94 @@ def test_block_full_data(
         post={},
         tx=tx,
     )
+
+
+@pytest.mark.valid_from("Prague")
+def test_block_full_access_list_and_data(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    intrinsic_cost: int,
+    total_cost_standard_per_token: int,
+    fork: Fork,
+):
+    """Test a block with access lists (60% gas) and calldata (40% gas) using random mixed bytes."""
+    import random
+    
+    attack_gas_limit = Environment().gas_limit
```

Comment:
On latest main we revised this approach and now this is the expected way of getting the maximum benchmarking gas:
```python
@pytest.mark.valid_from("Prague")
def test_block_full_access_list_and_data(
    state_test: StateTestFiller,
    pre: Alloc,
    intrinsic_cost: int,
    total_cost_standard_per_token: int,
    fork: Fork,
    gas_benchmark_value: int,  # New parameter, passed by the benchmark tests filler
):
    attack_gas_limit = gas_benchmark_value
```